From 509a5ee75b7efde73ff11386e51a2efe64bcbfa3 Mon Sep 17 00:00:00 2001
From: Raymond Wong <infiwang@pm.me>
Date: Fri, 31 Mar 2023 03:58:31 +0800
Subject: [PATCH 079/156] Revert "riscv(misc): FFI related bug workaround"

Fixed by c897659 "riscv(interp): reallocate TMP registers to comply with ABI"
---
 src/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile b/src/Makefile
index 9dd7dfc1..108825e4 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -36,7 +36,7 @@ CC= $(DEFAULT_CC)
 # to slow down the C part by not omitting it. Debugging, tracebacks and
 # unwinding are not affected -- the assembler part has frame unwind
 # information and GCC emits it where needed (x64) or with -g (see CCDEBUG).
-CCOPT= -O1 -fomit-frame-pointer
+CCOPT= -O2 -fomit-frame-pointer
 # Use this if you want to generate a smaller binary (but it's slower):
 #CCOPT= -Os -fomit-frame-pointer
 # Note: it's no longer recommended to use -O3 with GCC 4.x.
-- 
2.42.0

